GAE/src/com/gallatinsystems/survey/domain/SurveyGroup.java

```diff
@@ -166,15 +166,15 @@ public void setDefaultLanguageCode(String defaultLanuageCode) {
         this.defaultLanguageCode = defaultLanuageCode;
     }
 
```

Comment:
Shall we maybe remove commented out code?

Reply:
Certainly.